src/main/java/useCases/Timer/TimerInteractor.java

```diff
@@ -23,7 +19,7 @@ public class TimerInteractor {
     //constructor
     /**
      * Constructs TimerInputData object given the input duration
-     * @param inputDuration The duration for the Timer (how long of a Timer session do we want)
+     * @param inputDuration The duration for the Timer, how long of a Timer session do we want
      */
     TimerInteractor(Duration inputDuration){
```

Comment:
The constructor should take in a presenter that implements some interface as argument. The methods of this interactor should call the appropriate methods of the presenter that invoke the view change.

Reply:
I think this constructor should take in a RequestModel which takes in the Duration input from the person using this system, instead of directly taking Duration as a parameter.